x-pack/metricbeat/mbreceiver/receiver_test.go

```diff
@@ -29,16 +47,18 @@ func TestNewReceiver(t *testing.T) {
 					},
 				},
 			},
-			"output": map[string]interface{}{
-				"otelconsumer": map[string]interface{}{},
+			"output": map[string]any{
+				"otelconsumer": map[string]any{},
 			},
-			"logging": map[string]interface{}{
+			"logging": map[string]any{
 				"level": "debug",
 				"selectors": []string{
 					"*",
 				},
 			},
-			"path.home": t.TempDir(),
+			"path.home":    t.TempDir(),
+			"http.enabled": true,
+			"http.host":    monitorHost,
```

Comment:
Can you add these to `TestMultipleReceivers` as well? It spawns two receivers so if there is anything fishy going on with global state we might be able to catch it.

Reply:
done.